README.md

```diff
@@ -59,4 +59,10 @@ optional:
 
 -   **Warning**: If your project or this project is facing problems with dependencies, try 
     `npm audit fix` first please! The reason is because `npm audit` could have you running
-    code that changes your npm config instead of the local `package.json`.
\ No newline at end of file
+    code that changes your npm config instead of the local `package.json`.
+
+## Run on lambda
+-   To run this on lambda, you need to put the `app` folder, production `node_modules` 
+    folder, `.env`, `config.js`, `index.js`, `package-lock.json`, and `package.json` into a 
```

Comment:
As part of "defense in depth," can we have the Lambda environment provide any "secrets" instead of packaging up `.env`?

Reply:
Yes, I believe you can inject the "secrets" at run time, doing something like `ENV_NAME=secretKey npm start`. Or for Lambda Functions, you can either add them manually in the `Environment variables` section. Or using the secret manager. (I am not sure what is the name of the service on AWS, nor how to use it)